masterlist.yaml

```
@@ -15869,11 +15869,29 @@ plugins:
         itm: 122
 
   # Additional Clockwork
+  - name: 'ClockworkBugFixes.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
+    clean:
+      - crc: 0x6ED1EE37
+        util: 'SSEEdit v4.0.3'
   - name: 'ClockworkCourierCounterspell.esp'
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/47087/' ]
     clean:
       - crc: 0x6ABC5154
         util: 'SSEEdit v4.0.3'
+  - name: 'ClockworkInterestingInhabitants.esp'
+    clean:
+      - crc: 0x9E77EAE9
+        util: 'SSEEdit v4.0.3'
```

I get a different checksum, is this a future version of the plugin?

```suggestion
    clean:
      - crc: 0xC73837D9
        util: 'SSEEdit v4.0.4'
```